project/AppDependencies.scala

```diff
@@ -1,23 +1,18 @@
-import play.core.PlayVersion
-import play.sbt.PlayImport._
-import sbt.Keys.libraryDependencies
-import sbt._
+import sbt.*
 
```

Comment:
If you add //format: OFF to this file (see [here](https://github.com/hmrc/card-payment/blob/main/project/AppDependencies.scala#L3)) and retain the nice indentation/formatting, I'd appreciate it 😄

Reply:
Doing the above stops scalariform from reformatting the file